AudioLink/Scripts/AudioLink.cs

```diff
@@ -100,14 +102,18 @@ public class AudioLink : MonoBehaviour
         private double _elapsedTimeMSW = 0;
         private int    _networkTimeMS;
         private double _networkTimeMSAccumulatedError;
+#pragma warning disable 414 ///The private field is assigned but its value is never used
```

Comment:
Is this caused by the introduced if guards? If so, you should just guard this variable as well.

Reply:
If not and it's never used, remove it

Reply:
> Is this caused by the introduced if guards? If so, you should just guard this variable as well.

no this was happening before the if guards, it only happens if UDON is not defined so only in avatar projects
 #if guarding it gave me compiler errors
I will check this again

Reply:
@pema99 if I #if guard this with UDON so that it doesn't throw the error in avatar projects, then it won't be available for future non-UDON projects

Reply:
Ok, fair enough